email_login/src/components/PanelWrapper.js

```diff
@@ -425,8 +425,17 @@ class PanelWrapper extends Component {
         : this.state.values.usernameOrEmailAddress;
     socketClient.disconnect();
     this.stopCountdown();
+    const hasPIN = hasPin();
+    if (!hasPIN)
```

Comment:
Don't tell me that this change help to the cases when the app ask you for a PIN when you installed the app for the first time.

Reply:
Yes? It even creates the database